### PR TITLE
Linting CSS and Ether

### PR DESCRIPTION
**Pull request by toni-leigh**
## QA Steps

* sense check in the demo site

### CODE REVIEW

File: src/utils/css/css.js [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
-  textInactive: "text--inactive"
+  clearfix: 'clearfix',
+  input: 'common-input',
+  hidden: 'hidden',
+  unselectable: 'unselectable',
+  textTag: 'text--tag',
+  textBold: 'text--bold',
+  textItalic: 'text--italic',
+  textSecondary: 'text--secondary',
+  textInactive: 'text--inactive'
```
Comment (ianoxley) [inline src/utils/css/css.js]: Minor: should we sort these alphabetically?

Comment (toni-leigh) [inline src/utils/css/css.js]: I'm only doing linting in order not to confuse matters.

I agree, we should do this sort of thing but think it obfuscates the PR related changes.

**Comment by SeanArmstrong** [comment]
Probably should add some release notes. Possibly a Utils section needs adding for this

**Comment by SeanArmstrong** [comment]
Tested Tooltip for StyleElement and GroupCharacter for insertAt

**Comment by SeanArmstrong** [comment]
Tested ButtonToggle, Inputs and Pager for CSS

**Comment by SeanArmstrong** [comment]
Chrome, Safari, Firefox, Edge, IE 11

**Pull request merged**